index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="tool-box">
+        <div class="headder">
+            <h4 class="name">Welcome</h4>
+            <img class="icon" src="images/Union.png" alt="icon">
+        </div>
+    </section>
+    <!-- welcome_page -->
+    <section class="welcome_page">
+        <!-- welcome message -->
+        <div class="welcome_message">
+            <h1 class="greeting">Hey there. I'm Bobby</h1>
+            <h2 class="work">I am a software developer</h2>
+            <h3 class="service">
+                I can help you build a product , feature or website Look through some 
+                of my work and experience! If you like what you see and have a project 
+                you need coded, don’t hestiate to contact me.
```

Comment:
[ Required ] Please consider using a `p` tag to wrap this section, since this is multiline text it makes more sense to use a `p` tag rather than the `h3` tag as a wrapper.